app/src/main/java/com/brandon/forzenbook/view/composables/GeneralComposables.kt

```diff
@@ -76,7 +76,7 @@ fun BlackButton(
 ) {
     Box(
         modifier = Modifier
-            .padding(LocalSizeController.current.small)
+            .padding(LocalDimenController.current.small)
             .height(40.dp)
```

Comment:
Constant

Reply:
Created an object to hold composable constants for some of these. for ones relating to DP, I added them to the LocalDimens area so they can be later readjusted based on screen size easily.

let me know if this seems appropriate. I can move these into constants in the object fairly easily if its not clean to do it this way.